jsonnet/kube-prometheus/main.libsonnet

```diff
@@ -11,6 +11,8 @@ local prometheus = import './components/prometheus.libsonnet';
 
 local platformPatch = import './platforms/platforms.libsonnet';
 
+local promlib = import './lib/prometheus.libsonnet';
```

Comment:
let's name it `utils.libsonnet` as it isn't really related to prometheus deployment and has a potential of mistaking it with `components/prometheus.libsonnet`.

Reply:
done